experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -615,19 +660,15 @@ export interface Sequence<in out TTypes extends AllowedTypes> extends TreeField
 	 * @param sourceStart - The starting index of the range to move (inclusive).
 	 * @param sourceEnd - The ending index of the range to move (exclusive)
 	 * @param source - The source sequence to move items out of.
-	 * @throws Throws if the types of any of the items being moved are not allowed in the destination sequence or if the input indices are invalid.
-	 * @remarks
-	 * All indices are relative to the sequence excluding the nodes being moved.
+	 * @throws Throws if the types of any of the items being moved are not allowed in the destination sequence or if any of the input indices are not in the range [0, `list.length`).
```

Comment:
This seems to omit some throwing scenarios for `sourceEnd` < `sourceStart` (`sourceEnd` <= `sourceStart` ?)

Reply:
Clarified